templates/manx.html

```diff
@@ -22,89 +21,154 @@ <h2>Manx</h2>
+    <div class="is-flex is-flex-direction-column">
         <h3>Terminal</h3>
-        <div class="column" style="flex:100%;padding:15px;text-align: left">
-            <div id="filters">
-            <center>
-                <table class="ability-filter" frame=void rules=rows style="border-spacing:5px;width:100%">
-                    <tr>
-                        <td>
-                            <div class="select is-small">
-                                <select id="session-id" onchange="getAbilities();clearTerminal();getShellHistory(this);">
-                                    <option value="" disabled selected>Select a session</option>
-                                    {% for s in sessions %}
-                                        <option value="{{ s.id }}" data-paw="{{ s.info }}" data-platform="{{ s.platform }}" data-executor="{{ s.executors[0] }}">{{ s.id }} - {{ s.info }}</option>
-                                    {% endfor %}
-                                </select>
-                            </div>
-                        </td>
-                        <td>
-                            <div class="select is-small">
-                                <select id="tactic-filter" onchange="filterTechniques()">
-                                    <option value="" disabled selected>Select a tactic</option>
-                                </select>
-                            </div>
-                        </td>
-                        <td>
-                            <div class="select is-small">
-                                <select id="technique-filter" onchange="filterProcedures()">
-                                    <option value="" disabled selected>Select a technique</option>
-                                </select>
-                            </div>
-                        </td>
-                        <td>
-                            <div class="select is-small">
-                                <select id="procedure-filter" onchange="showProcedure()">
-                                    <option value="" disabled selected>Select a procedure</option>
-                                </select>
-                            </div>
-                        </td>
-                    </tr>
-                </table>
-            </center>
+        <div class="is-flex is-flex-direction-row is-justify-content-space-around">
+            <div class="select is-small">
+                <select id="session-id"
+                        x-model="selectedSessionID"
+                        x-on:change="getTactics()">
+                    <option value="" disabled selected>Select a session</option>
+                    <template x-for="s in sessions" :key="s.id">
+                        <option x-bind:value="s.id" x-bind:data-paw="s.info"
+                                x-bind:data-platform="s.platform"
+                                x-bind:data-executor="s.executors[0]"
```

Comment:
If manx is open while I start a new manx agent, I get this in the console:
![Screen Shot 2021-11-01 at 5 05 41 PM](https://user-images.githubusercontent.com/6034483/139742544-035c4b36-ce85-4005-ba4f-bed1d10af81e.png)

I think this line might be the culprit